### ISSUE TITLE
Ability to inspect markup/styling of note

### ISSUE THREAD

**Issue by gingerbeardman** [glushchenko/fsnotes#129]
Maybe this would be providing (hidden) access to the web inspector?

**Comment by gingerbeardman** [comment]
Initially it was to try to debug what was going wrong with the syntax highlighting. I think that's been fixed now so it's very low priority.

**Comment by nth-chile** [comment]
Is it possible with a Swift app? AFAIK there is no web inspector

**Comment by jeff-h** [comment]
Yes, it's possible & pretty trivial; the WKWebView's inspector can be enabled in Swift like so:

`self.webView.configuration.preferences.setValue(true, forKey: "developerExtrasEnabled")`
